packages/rn-tester/Podfile

```diff
@@ -32,8 +32,10 @@ def pods(options = {})
     path: @prefix_path,
     fabric_enabled: fabric_enabled,
     hermes_enabled: hermes_enabled,
+    flipper_configuration: USE_FRAMEWORKS ? FlipperConfiguation.disabled : FlipperConfiguation.enabled,
```

Comment:
typo?
Configuation -> Configuration?

Reply:
Thank you! I didn't notice it and then used autocompletion! :)